src/components/App.tsx

```diff
@@ -40,6 +42,10 @@ const App: React.FC = () => {
               path="/surveys/:surveyId/*"
               element={ <SurveyScreen/> }
             />
+            <Route
+              path="/groups/:groupId*"
```

Comment:
Should this include a slash also?

```suggestion
              path="/groups/:groupId/*"
```